packages/bot-web-ui/src/components/blockly-loading/blockly-loading.tsx

```diff
@@ -1,22 +1,20 @@
 import React from 'react';
 import { Loading } from '@deriv/components';
-import RootStore from 'Stores/index';
-import { connect } from 'Stores/connect';
+import { observer } from '@deriv/stores';
+import { useDBotStore } from 'Stores/useDBotStore';
 
-type TBlocklyLoadingProps = {
-    is_loading: boolean;
-};
+const BlocklyLoading = observer(() => {
+    const { blockly_store } = useDBotStore();
+    const { is_loading } = blockly_store;
+    return (
+        <>
```

Comment:
```suggestion
        is_loading && (
                <div className='bot__loading'>
                    <Loading />
                </div>
            )
```

Reply:
we will need this fragment as this div is the only returning element